jep/0000/README.adoc

```diff
@@ -0,0 +1,850 @@
+= JEP-0000: Publishing incremental commits as Maven releases
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 0000
+
+| Title
+| Publishing incremental commits as Maven releases
+
+| Sponsor
+| https://github.com/jglick[@jglick]
+
+| Status
+// Uncomment the appropriate line.
+//| Not Submitted :information_source:
+| Draft :speech_balloon:
+//| Deferred :hourglass:
+//| Accepted :ok_hand:
+//| Rejected :no_entry:
+//| Withdrawn :hand:
+//| Final :lock:
+//| Replaced :dagger:
+//| Active :smile:
+
+| Type
+| Process
+
+| Created
+| :bulb: Date :bulb:
+
+| JIRA
+| link:https://issues.jenkins-ci.org/browse/JENKINS-50686[JENKINS-50686]
+
+// Uncomment if there will be a BDFL delegate for this JEP.
+//| BDLF-Delegate
+//| :bulb: Link to github user page :bulb:
+
+| Requires
+| :bulb: IEP-9 :bulb:
+
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+
+== Abstract
+
+The Jenkins source base is spread across numerous GitHub repositories:
```

Comment:
Much of this abstract belongs in the motivation.  This section should be one paragraph that describe the proposal.   From what I can tell line 94 is about where the abstract could start, but it might be even line 99.  
Think of this as the elevator-pitch description of the proposed change.